addon/components/frost-table-body.js

```diff
@@ -51,10 +51,42 @@ export default Component.extend({
 
   // == Functions =============================================================
 
+  setCellWidths (position) {
+    const curBodyColumn = this.$(`.frost-table-row .frost-table-body-cell:nth-child(${position})`)
```

Comment:
As discussed previously, I'm okay with not using `eq` here since it doesn't apply in your case.